plugin.cpp

```diff
@@ -122,24 +122,23 @@
 static const char *default_config = QUOTE(
 	{
 		"plugin": {
-			"description": RULE_DESC,
+			"description": "Generate a notifiction if the values of one or all the configured assets exceeds a configured value",
 			"type": "string",
 			"default": RULE_NAME ,
 			"readonly": "true"
 		},
 		"description": {
-			"description": "Generate a notification if all configured assets trigger",
```

Comment:
@MarkRiddoch @pintomax  description key value pair is required for config item. Though value can be empty.

Reply:
description is not used and should not really have been there

Reply:
Either we should remove the `"description": ` config item itself. Or keep `"description": ` attribute for it.